l10n_ar_ux/__init__.py

```diff
@@ -4,15 +4,17 @@
 ##############################################################################
 from . import controllers
 from . import models
+from odoo import api
 from . import reports
 from . import wizards
 from .hooks import post_init_hook
 
 from odoo.addons.l10n_latam_invoice_document.models.account_move import AccountMove
 from odoo.exceptions import UserError
 
+from odoo.addons.l10n_ar.models.account_fiscal_position import AccountFiscalPosition
 
-def monkey_patch_inverse_l10n_latam_document_number():
+def monkey_patches():
     # monkey patch
     orginal_method = AccountMove._inverse_l10n_latam_document_number
 
```

Comment:
We need to ensure that the _inverse_l10n_latam_document_number is been called and is working.
please in a local environment add  a pdb or print and check that is working.
Functional you can go to vendor bills tru to create a new one with a use document our hase journal and when you try to set the document number this method should be called